reference/config_files/default_profile.rst

```diff
@@ -0,0 +1,519 @@
+    MyPath1=(path)/some/path11
+
+    # Prepend another PATH to "MyPath1"
+    MyPath1=+(path)/other path/path12
+
+    # Unset the variable "PATH" for all the packages matching the pattern "mypkg*"
+    mypkg*:PATH=!
+
+
+Then, the result of applying this profile is:
+
+* ``MyVar1``: ``My Value; other MyValue12``
+* ``MyPath1``:
+    * Unix: ``/other path/path12:/some/path11``
+    * Windows: ``/other path/path12;/some/path11``
+* ``mypkg*:PATH``: ``None``
+
+
+See more information about the new environments in the :ref:`reference_tools_env` reference.
+
+
+.. _profiles_runenv:
+
+[runenv]
+++++++++++
+
+Available since: `1.53.0 <https://github.com/conan-io/conan/releases/tag/1.53.0>`_
+
+.. important::
+
+    The use of this ``[runenv]`` section requires using the ``VirtualRunEnv`` generator in your recipe.
+
+This profile section allows defining environment variables that will be injected to the
+environment every time the ConanFile ``run(cmd, env="conanrun")`` method is invoked. You can use the same
+operators explained for the :ref:`profiles_buildenv` section and also define `PATH`
+variables.
+
+.. _profiles_tools_conf:
+
+Tools configurations
+--------------------
+
+.. important::
+
+    This feature is still **under development**, while it is recommended and usable and we will try not to break them in future releases,
+    some breaking changes might still happen if necessary to prepare for the *Conan 2.0 release*.
+
+Tools configurations can also be used in profile files and *global.conf* one. Profile values will have priority over globally defined ones in *global.conf*, and can be defined as:
+
+.. code-block:: text
+
+    [settings]
+    ...
+
+    [conf]
+    tools.microsoft.msbuild:verbosity=Diagnostic
+    tools.microsoft.msbuild:max_cpu_count=2
+    tools.microsoft.msbuild:vs_version = 16
+    tools.build:jobs=10
+
+.. seealso::
+
+    You can see more information about configurations in :ref:`global.conf section <reference_config_files_global_conf>`.
+
+
+Profile composition
+-------------------
+
+You can specify multiple profiles in the command line. The applied configuration will be the composition
+of all the profiles applied in the order they are specified.
+
+If, for example, you want to apply a :ref:`tool require<reference_conanfile_attributes_build_requires>`, like a ``cmake``
+installer to your dependency tree, it won't be very practical adding the `cmake` installer reference, e.g,
+``cmake/3.16.3`` to all your profiles where you could need to inject ``cmake`` as a tool require.
+
+You can specify both profiles instead:
+
+.. code-block:: text
+   :caption: *[CONAN_HOME]/profiles/cmake_316*
+
+   [tool_requires]
+   cmake/3.16.3
+
+.. code-block:: bash
+
+   $ conan install . --profile clang --profile cmake_316
+
+Profile includes
+----------------
+
+You can include other profiles using the ``include()`` statement. The path can be relative to the current profile, absolute, or a profile
```

Comment:
in this case, it's kinda unclear how do nested `include()` statements work with relative paths, e.g. we have:
```
cat a.profile
include(../b.profile)

cat b.profile
include(../c.profile)
```
will it resolve to `../../c.profile` or just `../c.profile`?